src/nav/ModalNavBar.js

```diff
@@ -32,7 +32,18 @@ class ModalNavBar extends PureComponent<Props> {
     ];
 
     return (
-      <View style={[contextStyles.navBar]}>
+      <View
+        style={[
+          {
+            borderColor: 'hsla(0, 0%, 50%, 0.25)',
+            flexDirection: 'row',
+            height: NAVBAR_SIZE,
+            alignItems: 'center',
+            borderBottomWidth: 1,
+            backgroundColor: contextStyles.navBar.backgroundColor,
```

Comment:
This looks a bit funny because we're inlining `contextStyles.navBar`... but keep using one piece of `contextStyles.navBar` :-)

There's another style `contextStyles.backgroundColor`, which consists exactly of the same `backgroundColor` property found in `contextStyles.navBar`. So this and the other similar site could use that instead.

Then that also makes it possible to actually remove the `navBar` definition in this same commit. That makes the change a bit simpler to understand, because (a) the existing definition and the inlined versions are both present in the diff (in red `-` and green `+` respectively), and (b) it becomes clear this is inlining all the uses.

Still good to have a second, followup commit (just like this version of the branch does) to switch from `contextStyles.backgroundColor` to `this.context.backgroundColor` and the new context API.

Reply:
> There's another style `contextStyles.backgroundColor`, which consists exactly of the same `backgroundColor` property found in `contextStyles.navBar`. So this and the other similar site could use that instead.

Mm, I missed this in your previous review; I'll use this. It seems like it's actually spelled `contextStyles.backgroundColor.backgroundColor`, which is weird but doable until the next commit lands. 🙂